call_handlers/debugtools_commands.lua

```diff
@@ -618,4 +618,22 @@ function Commands:print_ai_stack_command(session, response, entity)
    response:resolve({'reported traceback for entity '..tostring(entity)})
 end
 
+function Commands:add_conversation_subject_command(session, response, entity, subject, polarity)
+   local conversation_component = entity:add_component('stonehearth:conversation')
+   local subject = conversation_component:add_conversation_subject(subject, polarity)
+   response:resolve(subject)
+end
+
+function Commands:get_conversation_subject_command(session, response, entity, subject)
+   local conversation_component = entity:add_component('stonehearth:conversation')
```

Comment:
The getters should probably check if the component exists before calling a function on it instead of automatically adding the component. Otherwise players may add the conversation component to any selected entity if they call this on it.